src/darwin/Framework/CHIP/MTRDeviceController.mm

```diff
@@ -162,6 +168,11 @@ - (void)cleanup
         delete _pairingDelegateBridge;
         _pairingDelegateBridge = nullptr;
     }
+
+    if (_partialDACVerifier) {
```

Comment:
Please make sure deallocation order in cleanup is the reverse of allocation order in initWithFactory.  So this should be deleted before `_pairingDelegateBridge`.

Reply:
Done.